### ISSUE TITLE
'Fragment' is not defined 

### ISSUE THREAD

**Issue by budarin** [eslint/eslint#9675]
package.json
```
"babel-core": "^6.26.0",
"babel-preset-react": "^6.24.1",
"eslint": "4.12.1",
"babel-eslint": "^8.0.3",
"react": "^16.2.0",
"react-dom": "^16.2.0",
```

In code 
```jsx
return (
    <Fragment>
           ... code
    </Fragment>
);
```
here is the error:
```Error:(9, 10) ESLint: 'Fragment' is not defined. (react/jsx-no-undef)```

**Comment by ilyavolodin** [comment]
The error is reported by the rule in eslint-plugin-react. We do not maintain that plugin. You probably want to file the issue there.

**Comment by budarin** [comment]
Thanks for pointing!

**Comment by dcousens** [comment]
Related issue https://github.com/yannickcr/eslint-plugin-react/issues/1579

**Comment by mattiascaricato** [comment]
you have to inlude it 
```javascript
import React, { Fragment } from 'react';
```

**Comment by budarin** [comment]
I use to, but I want to use <> syntax :)
